docs/src/literate-tutorials/heat_equation_hdiv.jl

```diff
@@ -0,0 +1,234 @@
+# # [Heat equation - Mixed H(div) conforming formulation)](@id tutorial-heat-equation-hdiv)
+# As an alternative to the standard formulation for solving the heat equation used in
+# the [heat equation tutorial](@ref tutorial-heat-equation), we can used a mixed formulation
+# where both the temperature, $u(\mathbf{x})$, and the heat flux, $\boldsymbol{q}(\boldsymbol{x})$,
+# are primary variables. From a theoretical standpoint, there are many details on e.g. which combinations
+# of interpolations that are stable. See e.g. [Gatica2014](@cite) and [Boffi2013](@cite) for further reading.
+# This tutorial is based on the theory in
+# [Fenics' mixed poisson example](https://fenicsproject.org/olddocs/dolfin/1.4.0/python/demo/documented/mixed-poisson/python/documentation.html).
```

Comment:
Yes, I was struggling with that today. Not sure if that's just a mistake though, since it is still linked to from their GitHub webpage? 

https://github.com/FEniCS/docs

Reply:
Probably forgot to pay the bill. Perhaps we should buy it and redirect it to Ferrite :)

Reply:
Seems to be back?